src/bin/server/mod.rs

```diff
@@ -42,20 +45,27 @@ pub fn serve(
     let transcoder_addr: Addr<Transcoder> = transcoder.start();
 
     server::new(move || {
-        App::with_state(AppState::new(
+        let app = App::with_state(AppState::new(
             gql_executor.clone(),
             transcoder_addr.clone(),
             pool.clone(),
         )).resource("/graphql", |r| r.method(http::Method::POST).with(graphql))
-            .resource("/", |r| r.method(http::Method::GET).f(graphiql))
+            .resource("/graphiql", |r| r.method(http::Method::GET).f(graphiql))
             .register_transcode_handler(TranscodeTarget::MP3V0)
             .register_transcode_handler(TranscodeTarget::AACV5)
             .resource(&Song::get_raw_stream_url("{id}"), |r| {
                 r.method(http::Method::GET).with(streaming::song_handler)
             })
             .resource(&Album::get_artwork_url("{id}"), |r| {
                 r.method(http::Method::GET).with(streaming::artwork_handler)
-            })
+            });
+
+        #[cfg(feature = "embed_web")]
```

Comment:
Could use a similar pattern to `register_transcode_handler` instead. Not a big deal. What do you think?